src/index.html

```diff
@@ -7,6 +7,39 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--container calendar--start-cell_sun calendar--month-length_29">
```

Comment:
Overloaded. You don't need `calendar--container`.
Also read task carefully about modifier names.